src/main/java/lotto/model/Result.java

```diff
@@ -12,10 +11,10 @@
 public class Result {
     private Map<Rank, Integer> matchResult;
 
-    public Result(Lottos lottos, Lotto winLotto) {
+    public Result(Lottos lottos, WinningLotto winLotto) {
         this.matchResult = createResultMap();
```

Comment:
```suggestion
        this.matchResult = Rank.createWinningRanks();
```

이렇게 바꿔도 되지 않나요?
Rank 값을 꺼내서 각각 0으로 초기화 해줄게 아니라 Rank 내부에서 필요한 값 셋팅해서 필요한 값으로 반환해주면 재사용도 가능하지 않을까요?

Reply:
물론 createWinningRanks 메서드는 Map을 반환하도록 수정하셔야합니다.